### PR TITLE
Fix admin cyro topic

### LINKED ISSUE #1590: Runtime with admin verb cryo
## Testmerges:

Round ID: 474
BYOND Version: 512.1464
Server revision compiled on: 
Master commit: 5c0c4281810834ea8a80f7a6ea69464c6f35ad4c
The following pull requests are currently test merged:
#1577: 'Runner balance tweaks' by Rohesie at commit cff6a9abc4
#1578: 'Xenos now break barrels properly.' by DominikPanic at commit e5729fcae4
#1580: 'Gives CLF leader ammo.' by DominikPanic at commit ed36f0c78b
#1556: 'adding tox bottles to med vendors' by Ghommie at commit 4c9591202f
#1568: 'Fixes a race condition init runtime' by DominikPanic at commit 524ae62dee
#1561: 'Fixes an ancient init runtime' by DominikPanic at commit 1ea06746e4
#1563: 'Fixes evil emote runtime' by DominikPanic at commit 51d4ef746c
#1564: 'Possibly fixes another runtime' by DominikPanic at commit cb3dd612ac
#1552: 'Fixes otherwordly sound spamming.' by Ghommie at commit c9c84c5f0f
#1540: 'Ports SSinput from /tg/' by DominikPanic at commit d44234fc0b

## Reproduction:

Cryoing a SSD Ship Engineer buckled to a chair.

```
[05:15:13] Runtime in radio.dm, line 157: Cannot execute null.get spans().

proc name: talk into (/obj/item/radio/talk_into)
usr: Rohesie/(Lydia 'Synth' Tournier)
usr.loc: (Vehicle Armor Storage (64, 80, 3))
src: the station bounced radio (/obj/item/radio)
src.loc: Luka Vega (/mob/living/carbon/human)
call stack:
the station bounced radio (/obj/item/radio): talk into(null, "Luka Vega has entered long-ter...", 1461, null, null)
Luka Vega (/mob/living/carbon/human): despawn(null, "Engineering")
Luka Vega (/mob/living/carbon/human): despawn(null, "Requisitions")
rohesie\'s admin datum (Projec... (/datum/admins): Topic("src=\[0x21000a2c];admin_token=...", /list (/list))
Rohesie (/client): Topic("src=\[0x21000a2c];admin_token=...", /list (/list), rohesie\'s admin datum (Projec... (/datum/admins))
Rohesie (/client): Topic("src=\[0x21000a2c];admin_token=...", /list (/list), rohesie\'s admin datum (Projec... (/datum/admins))
```

### PR DESCRIPTION
**Pull request by psykzz**
Fixes #1590

### CODE REVIEW

File: code/game/machinery/cryopod.dm [hunk 1]
```diff
@@ -244,7 +244,7 @@
 /mob/proc/despawn(obj/machinery/cryopod/pod, dept_console = CRYO_REQ)
 
 	//Handle job slot/tater cleanup.
-	if(job)
+	if(job && job != ROLE_XENOMORPH && job != ROLE_XENO_QUEEN && job != ROLE_SURVIVOR && job != ROLE_ERT)
```
Comment (DominikPanic) [inline code/game/machinery/cryopod.dm]: Use job in ROLES_REGULAR_ALL

File: code/game/machinery/cryopod.dm [hunk 2]
```diff
@@ -244,7 +244,7 @@
 /mob/proc/despawn(obj/machinery/cryopod/pod, dept_console = CRYO_REQ)
 
 	//Handle job slot/tater cleanup.
-	if(job)
+	if(job && job in JOBS_REGULAR_ALL)
```
Comment (spookydonut) [inline code/game/machinery/cryopod.dm]:
```suggestion
	if(job in JOBS_REGULAR_ALL)
```

**Comment by spookydonut** [comment]
Needs a valid description and changelog entry

**Comment by Rohesie** [comment]
We don't really changelog every small tweak, we tend to reserve that for things noticeable by the players (new features, balance changes, stuff like that).
Else our changelog would be much, much fatter.
Feel free to bring this up in #dev-general and we may change our practices in the future, if everyone agrees, but it's not a standard we've been following so far.

**Comment by psykzz** [comment]
If there was a change to log it would make sense.
This just fixes expected behaviour.

**Pull request merged**